src/wagtail_ai/text_splitters/base.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABCMeta, abstractmethod
+from typing import TypeVar
+
+TextSplitterLengthConfig = TypeVar("TextSplitterLengthConfig")
+
+
+class BaseTextSplitterLength(metaclass=ABCMeta):
```

Comment:
**suggestion:** Could this be a Protocol to align with `TextSplitterProtocol`? Feels like this could also be named more descriptively, although I'm not entirely sure how. Something like `TextSplitterLengthCalculator`?